From 0957164b45af9b1532dd93989d6631c3e2bc9413 Mon Sep 17 00:00:00 2001
From: Tzafrir Cohen <tzafrir.cohen@xorcom.com>
Date: Sun, 24 Feb 2013 20:29:05 +0200
Subject: [PATCH] Consider linux-gnuspe as linux-gnu

The powerpcspe Linux port uses linux-gnuspe as the OS string.
Our build system shouldn't really care for that, so just call
it linux-gnu.

Also: changing "gnueabi" to "gnueabi*" as is in later versions of Upstream.
This also handles "gnueabihf".
---
 configure.ac |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index 5a01fc2..e723c0f 100644
--- a/configure.ac
+++ b/configure.ac
@@ -172,7 +172,7 @@ case "${host_os}" in
      OSARCH=cygwin
      PBX_WINARCH=1
      ;;
-     linux-gnueabi)
+     linux-gnueabi* | linux-gnuspe)
      OSARCH=linux-gnu
      ;;
      *)
-- 
1.7.10.4

